node/overseer/src/lib.rs

```diff
@@ -17,7 +17,7 @@
 //! # Overseer
 //!
 //! `overseer` implements the Overseer architecture described in the
-//! [implementers-guide](https://github.com/paritytech/polkadot/blob/master/roadmap/implementers-guide/guide.md).
+//! [implementers-guide](https://w3f.github.io/parachain-implementers-guide/node/index.html).
```

Comment:
I guess this has become the canonical source of the guide lately. It's as stable as the current location in the source, so no complaint against this change.